tools/build_node.js

```diff
@@ -41,23 +41,18 @@ async function buildNodeHighlightJS() {
 }
 
 async function buildPackageJSON() {
-  const CONTRIBUTOR = /^- (.*) <(.*)>$/
-
-  let authors = await fs.readFile("AUTHORS.txt", {encoding: "utf8"})
-  let lines = authors.split(/\r?\n/)
-  let json = require("../package")
-  json.contributors = lines.reduce((acc, line) => {
-    let matches = line.match(CONTRIBUTOR)
-
-    if (matches) {
-      acc.push({
-        name: matches[1],
-        email: matches[2]
-      })
-    }
-    return acc;
-  }, []);
-  await fs.writeFile(`${process.env.BUILD_DIR}/package.json`, JSON.stringify(json, null, '   '));
+  const CONTRIBUTOR = /^- ([^<(]+( ?<.+>)?( ?\(http.+\))?)/;
```

Comment:
Would \s+ be inadequate here at the beginning?

Reply:
N/m, but hmmmmm... will `.*` or `.*?` not get the job done equally well?

Reply:
Where exactly?
The format in the authors file is stable, so don't have to be too flexible.

Reply:
Maybe I'm just being pedantic...  the character group is bugging me if a simpler .* does the job.

```
/^- (.*?(<.+>)?\s*(\(http.+\))?)$/
```

Cleaner, no?

Reply:
The problem is that the authors file contains lines like this
```
- Josh Goebel <hello@joshgoebel.com> (maintainer)
```
where we don't want the part in parens.

See https://regexr.com/5ipfo for testcases

Reply:
Yeah... then .* can be too greedy in some cases.  I will resolve. :)